pkg/api/addons_test.go

```diff
@@ -3428,4 +3507,13 @@ func getDefaultAddons(version string) []KubernetesAddon {
 			},
 		},
 	}
+
+	if common.IsKubernetesVersionGe(version, "1.15.0") {
```

Comment:
Yay w/ the new addons UT interfaces we just add default addons in one place and get that coverage everywhere